content/200-concepts/150-data-platform/08-billing/02-plans-and-quotas.mdx

```diff
@@ -0,0 +1,142 @@
+
+|                                               | `Early Adopter`                                                  | `Free`                                                   | `Scale`                                                                                                                                                                   | `Enterprise`                                 |
+| --------------------------------------------- | ---------------------------------------------------------------- | -------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------- |
+| Team members                                  | 10                                                               | 3                                                        | Unlimited<br /><br /> <strong>Quota</strong><br /><br />5 <br /><br /><strong>Overage</strong><br /><br />Each additional member at a fixed price                         | Custom                                       |
+| Member roles                                  | • Admin<br />• Viewer<br />• Developer<br />• Collaborator       | Admin only                                               | • Admin<br />• Viewer<br />• Developer<br />• Collaborator                                                                                                                | • Admin<br />• Viewer<br />• Developer<br /> |
+| Environments                                  | 7                                                                | 2                                                        | Unlimited<br /><br /> <strong>Quota</strong><br /><br />5 <br /><br /><strong>Overage</strong><br /><br />Each additional member at a fixed price                         | Custom                                       |
+| **Data Proxy**<br /><br /> - CPU usage        | 100,000,000<br /><br />CPU&nbsp;ms<br /><br />(hard limit)       | 3,000,000<br /><br />CPU&nbsp;ms<br /><br />(hard limit) | Unlimited<br /><br /><strong>Quota</strong><br /><br />30,000,000 CPU&nbsp;ms<br /><br /><strong>Overage</strong><br /><br />Each additional CPU&nbsp;ms at a fixed price | Unlimited<br /><br />Custom pricing          |
+| **Data Proxy**<br /><br /> -&nbsp;Concurrency | • 3 QEIDs\* per project<br />• 5 QEIDs\* per account/team member | 100 QEIDs\* per project                                  | 100 QEIDs\* per project                                                                                                                                                   | Custom                                       |
+
+<Admonition type="warning">
+
+\*_**QEIDs**_ refers to the number of available query engines that the Prisma Data Proxy can start and keep alive to manage the connection pool to the database for a project.
```

Comment:
Let's keep this for now, but I am not sure this is great to have. It leaks the abstraction that we use internally and users should not have to think of it at this low level. 

The only way I think of to phrase this better would be to frame it "limited concurrency" or something along those lines.

Reply:
Ok, I've rephrased this remove the QEID concreteness and used High and Low for concurrency in the various plans.